packages/big-design-icons/scripts/build-flags.js

```diff
@@ -27,20 +28,29 @@ async function convertToReactComponent(filePath, iconName) {
   const svgCode = await readFile(filePath, 'utf8');
   const destPath = join(DEST_PATH, `${iconName}.tsx`);
 
-  const code = await svgr(
+  const code = await transform(
     svgCode,
     {
       ...config,
       // Need to add the svgo config here to have access to iconName
       svgoConfig: {
         plugins: [
           {
-            prefixIds: {
+            name: 'preset-default',
+            params: {
+              overrides: {
+                removeViewBox: false,
+              },
+            },
+          },
+          {
+            name: 'prefixIds',
+            params: {
               prefix: iconName,
             },
-            removeViewBox: false,
-            removeXMLNS: true,
           },
+          'removeXMLNS',
+          removeInkscapeAttrs,
```

Comment:
This is only used within `flags` as the third-party library we use doesn't strip these attributes out. TS wouldn't build the icons with these attributes left behind, as it's unknown to them and they are unnecessary to use.